### ISSUE TITLE
Public Hosting

### ISSUE THREAD

**Issue by ehmenez** [cobisja/tad-php#43]
Hi!

I woul like to know if the library works on a public hosting?

Thanks.

**Comment by cobisja** [comment]
Greetings!,

Actually I can not tell if TAD-PHP works in a public hosting because it was
developed for providing a solution for a specific environment (a private
network). However I think it should work if your device is public
accessible from internet. Remember, TAD-PHP uses curl for sending command
and receiving responses, and also it uses UDP protocol, where both of them
needs an IP address for reaching the device.

So, you have to test the library on a public hosting.

Best regards,

Cobis

2018-02-03 13:13 GMT-04:00 ehmenez <notifications@github.com>:

**Comment by send2dawood** [comment]
Yes it works but you need a pubic ip and some setting on your dsl router.

**Comment by sangse0911** [comment]
Sorry, when i use library in public hosting, it only works with soap_port 80, whenever change the different port and turn off port 80, it cannot connect and get data, you can help me to fix it ?, thank

**Comment by mdestafadilah** [comment]
> Sorry, when i use library in public hosting, it only works with soap_port 80, whenever change the different port and turn off port 80, it cannot connect and get data, you can help me to fix it ?, thank

any update, resolve your problem?
